cpp/src/DataStorm/Instance.cpp

```diff
@@ -19,6 +19,12 @@ using namespace Ice;
 Instance::Instance(CommunicatorPtr communicator, function<void(function<void()> call)> customExecutor)
     : _communicator(std::move(communicator))
 {
+    if (_communicator->getDefaultObjectAdapter() != nullptr)
```

Comment:
This wont catch some cases where the application set a default OA before starting the Node. For those we need to relay in the documentation.